tools/release/lib/release.py

```diff
@@ -1,201 +1,21 @@
 #!/usr/bin/env python3
 
 import re
-from typing import Optional, List, Set, Union, Type
-from github.PullRequest import PullRequest
-from github.GitRelease import GitRelease
-from github.Repository import Repository
+from github import Repository, GitRelease
 
 
 def is_valid_tag(tag: str) -> bool:
     tag_regex = r'^v\d+\.\d+\.\d+$'
     return bool(re.match(tag_regex, tag))
 
 
-def create_release(repository: Repository, tag: str, notes: str) -> Union[None, Type[Exception]]:
+def create_release(repository: Repository, tag: str) -> GitRelease:
```

Comment:
I believe this would be: `GitRelease.GitRelease`

Reply:
It seems `from github import Repository, GitRelease` also works :-)